doc/releases/releases.yml

```diff
@@ -12,6 +12,12 @@
 # If a version might represent an actual number (e.g. 0.80) quote it.
 #
 releases:
+  squid:
+    target_eol: 2026-XX-XX
```

Comment:
@neha-ojha: I'll alter this when we have a target EOL.

Reply:
Changed it to tomorrow since I'm pretty sure we're going to release then.